README.md

```diff
@@ -1,10 +1,31 @@
 # NullAuthenticator
 
+[![PyPI](https://img.shields.io/pypi/v/nine.svg)](https://github.com/jupyterhub/nullauthenticator)
+
+
 Null Authenticator for JupyterHub instances that should have no login mechanism,
 such as those that exclusively allow access via API token.
 
-Enable null authenticator:
+## Installation
+
+As prerequisites, Python 3.4+ and JupyterHub 0.8.0+ are required to use Null
```

Comment:
NullAuthentictator works on 0.7 too!

Reply:
Good to know @yuvipanda. The `requirements.txt` specifies 0.8 so that's what I used. @minrk your call on what you wish to "officially" support :smile:

Reply:
Ah, ok! I saw https://github.com/jupyterhub/nullauthenticator/blob/master/nullauthenticator.py#L26 which was put in explicitly to support 0.7 in some form, although it'll be quite awkward. Am happy for this to only support 0.8! We should probably remove that 0.7 shim at some point though (not in this PR!)

Reply:
Let's call it 0.8. While the Authenticator API itself happens to work with 0.7, the fact that token auth, etc. don't until 0.8 means that it's not useful.